stan/math/rev/fun/elt_multiply.hpp

```diff
@@ -57,7 +58,46 @@ auto elt_multiply(const Mat1& m1, const Mat2& m2) {
     });
     return ret_type(ret);
   }
-}  // namespace math
+}
+
+/**
+ * Return specified matrix multiplied by specified scalar where at least one
+ * input has a scalar type of a `var_value`.
+ *
+ * @tparam T1 type of the scalar
+ * @tparam T2 type of the matrix or expression
+ *
+ * @param A scalar
+ * @param B matrix
+ * @return product of matrix and scalar
+ */
+template <typename T1, typename T2, require_not_matrix_t<T1>* = nullptr,
+          require_matrix_t<T2>* = nullptr,
+          require_any_st_var<T1, T2>* = nullptr,
+          require_not_row_and_col_vector_t<T1, T2>* = nullptr>
+inline auto elt_multiply(const T1& A, const T2& B) {
+  return multiply(A, B);
+}
```

Comment:
Should the prim impl of these just divert to multiply so we have consistent behavior everywhere?